jenkins-design-language/less/components/jtable.less

```diff
@@ -105,10 +105,16 @@ a.JTable-row--href {
 }
 
 .JTable-cell-contents {
-    width:100%;
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+    .text-overflow();
+
+    > code,
```

Comment:
Which component means we need `> code` here? Won't this complicate things if we want to put wrapping blocks of logs in a table some time in the future?

Reply:
That is for CommitID (used in the commit column), I will add an extra class there so the css rule is not using the tag name

Reply:
I didn't actually need to make another class, I just used the one I use in the message column so everything should be ok now